pwiz_tools/Skyline/Model/SrmDocument.cs

```diff
@@ -1544,6 +1555,17 @@ public SrmDocument ImportMassList(MassListInputs inputs,
                 {
                     errorList.Add(new TransitionImportErrorInfo(x));
                 }
+                catch (Exception ex)

```

Comment:
I don't think this is the right place to put this. This is supposed to be for issues with parsing a line, not for global errors. You should be catching other exceptions higher up the callstack.

Reply:
Catching it here is advantageous as it ties into the reporting mechanism for both the UI and commandline use cases.